src/js/components/charts/Chart.js

```diff
@@ -96,7 +97,11 @@ var Chart = createReactClass({
   render() {
     // at the moment, 'chart' is used to inject the chart color palette.
     // we should reclaim it as the rightful className of <Chart />
-    return <div className="chart-chart">{this.getChildren()}</div>;
+    return (
+      <div ref="chart" className="chart-chart">
```

Comment:
string based refs are deprecated. Shall we switch to `React.createRef()`?

Reply:
I was under the impression `React.createRef()` only works in a class scenario, which we can't go to due to the mixins, am I mistaken? Could you point me to articles / docs for that?

Reply:
then we should use `callback` syntax instead `ref={el => (this.chartRef = el)}`

Reply:
Which will fail at the linter since lambdas are not allowed in props we should create a function on the object which we call.

Reply:
that could also be an option

Reply:
I think this is an exception for the linting rule